deployer.sh

```diff
@@ -118,6 +118,50 @@ echo "# Using inventory: "
 cat ${INVENTORY_PATH}
 echo "#######################################################################"
 
+
+if [ "${INSTALL_PROMETHEUS}" == "true" ]; then
+    # We're installing Prometheus, this means we have to connect to all
+    # nodes on the cluster to make sure the iscsi initator name is set correctly
+    # and to collect the initator names so we can create the iscsi LUN.
+    echo "Setting initator names..."
+
+    INAME="iqn.1994-05.com.redhat:${NAME_PREFIX}-master001"
+    INITIATORS="${INAME}"
+    sshpass -p"${ROOT_PASSWORD}" ssh ${SSH_ARGS} "root@${MASTER_HOSTNAME}" "echo InitiatorName=${INAME} > /etc/iscsi/initiatorname.iscsi; systemctl restart iscsi.service"
+
+    NODE_NUMBER=0;
+    for IP in ${INFRA_IPS}; do
+        printf -v NODE_NUMBER_PADDED "%03d" ${NODE_NUMBER}
+        INAME="iqn.1994-05.com.redhat:${NAME_PREFIX}-infra${NODE_NUMBER_PADDED}"
+        INITIATORS="${INITIATORS} ${INAME}"
+        sshpass -p"${ROOT_PASSWORD}" ssh ${SSH_ARGS} "root@${IP}" "echo InitiatorName=${INAME} > /etc/iscsi/initiatorname.iscsi; systemctl restart iscsi.service"
+        let "NODE_NUMBER++"
+    done
+
+    NODE_NUMBER=0;
+    for IP in ${COMPUTE_IPS}; do
+        printf -v NODE_NUMBER_PADDED "%03d" ${NODE_NUMBER}
+        INAME="iqn.1994-05.com.redhat:${NAME_PREFIX}-compute${NODE_NUMBER_PADDED}"
+        INITIATORS="${INITIATORS} ${INAME}"
+        sshpass -p"${ROOT_PASSWORD}" ssh ${SSH_ARGS} "root@${IP}" "echo InitiatorName=${INAME} > /etc/iscsi/initiatorname.iscsi; systemctl restart iscsi.service"
+        let "NODE_NUMBER++"
+    done
+
+    echo "Initiators: ${INITIATORS}"
+    echo
+    echo "Creating iscsi LUN..."
+
+    ISCSI_LUN_ID=$(python "${WORKSPACE}/lun_manager.py" --server="${NETAPP_SERVER}" \
+                                                        --user="${NETAPP_USER}" \
+                                                        --name="cm-${NAME_PREFIX}" \
+                                                        --volume="${NETAPP_VOLUME}" \
+                                                        --vserver="${NETAPP_VSERVER}" \
+                                                        --size="15GB" \
```

Comment:
size should also be parameterized as we may want to enlarge the size when deploying a larger cluster